sql/sqlserver/db.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Vivek Menezes (vivek@cockroachlabs.com)
+
+package sqlserver
+
+import (
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/cockroachdb/cockroach/client"
+	"github.com/cockroachdb/cockroach/sql/sqlwire"
+	"github.com/cockroachdb/cockroach/util"
+)
+
+var allowedEncodings = []util.EncodingType{util.JSONEncoding, util.ProtoEncoding}
+
+var allPublicMethods = map[string]sqlwire.Method{
```

Comment:
The term public made sense in the KV api. Here I would: `s/allPublicMethods/allMethods/g`.